src/forms/forms.response.ts

```diff
@@ -102,6 +123,19 @@ class BaseQuestion {
     @ApiProperty({ example: null })
     optionGroup: OptionGroup;
 
+    @Optional()
+    @ApiProperty({
+        examples: {
```

Comment:
Would we revert this to example for now?

Reply:
yes, I just made the change to them now.